src/System.IO.Compression.Brotli/README.md

```diff
@@ -0,0 +1,214 @@
+﻿# System.IO.Compression.Brotli
+
+## Introduction
+
+Brotli is a generic-purpose lossless compression algorithm that compresses data
+using a combination of a modern variant of the LZ77 algorithm, Huffman coding
+and 2nd order context modeling, with a compression ratio comparable to the best
+currently available general-purpose compression methods. It is similar in speed
+to deflate but offers more dense compression.
+
+The specification of the Brotli Compressed Data Format is defined in [RFC 7932](https://www.ietf.org/rfc/rfc7932.txt).
+
+Brotli encoding is supported by most web browsers, major web servers, and some CDNs.
```

Comment:
Can you define what CDN stands for as this is the first time it is used in the README?